src/main/java/seedu/duke/parsers/ListParser.java

```diff
@@ -10,8 +10,8 @@
 public class ListParser extends Parser {
     private static final String LIST_ARGUMENT = StringConstants.LIST_ARGUMENT;
     //Unescaped Regex for testing:
-    //\s*(\"(?<listArgument>\w*)\")*\s*
-    private static final String LIST_FORMAT = "\\s*(\\\"(?<listArgument>\\w*)\\\")*\\s*";
+    //\s*((?<listArgument>\w*))*\s*
+    private static final String LIST_FORMAT = "\\s*((?<listArgument>\\w+))*\\s*";
```

Comment:
im not sure if the \s* in the front and back is needed here if the arguments will be trimmed either way